### PR TITLE
Bumped to latest version of Go 1.19

### PR DESCRIPTION
**Pull request by elle-Bee**

### CODE REVIEW

**Review by DelusionalOptimist — COMMENTED** [review]
Thanks @elle-Bee! Looks good.
Let's also update the go version at https://github.com/JIITODC/loadbalancer/blob/master/.github/workflows/ci.yaml#L13 and https://github.com/JIITODC/loadbalancer/blob/master/Backend/Dockerfile#L1.

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/Shreyas220/loadbalancer
 
-go 1.16
+go 1.19
```
Comment (rexagod) [inline go.mod:3]: You'd have to update the manifests using `go mod tidy`. This would update the `go.mod` the latest SHA sums.

**Review by rexagod — CHANGES_REQUESTED** [review]
Also, once @DelusionalOptimist's comments are addressed, the CI would trigger letting us know if things are working as expected. Please make sure that all Makefile targets are working as expected as well at that point.

File: go.mod (hunk 1 above)
Comment (elle-Bee) [reply go.mod:3]: Made the changes to the docker and yaml files and also updated the manifests. Can you please check it. Thanks!

**Review by DelusionalOptimist — APPROVED** [review]
Looks good to me! :+1:

**Comment by rexagod** [comment]
I can't verify if every target is working as expected right now, but I'd recommend doing so before merging to avoid regressions.

**Comment by elle-Bee** [comment]
unknown
@rexagod Verified the working and also attached the screenshot

**Review by rexagod — APPROVED** [review]

**Comment by DelusionalOptimist** [comment]
Thank you @elle-Bee! Looking forward to more contributions :smile:

**Comment by elle-Bee** [comment]
Looking forward to contributing even more to projects and thank you @DelusionalOptimist for being patient with me :)

**Pull request merged**